### PR TITLE
chore: Update info in Current LTS releases

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

File: 302.Release-information/01.Release-schedule/docs.md [hunk 1]
```diff
@@ -13,12 +13,11 @@ The lifecycle of an LTS release begins on the release date and ends one year lat
 
 <!-- the version number includes the patch release here, to get picked up by autoversion, but will be a minor version in the result -->
 <!--AUTOVERSION: "LTS releases: %"/lts -->
-At this time, we support the following LTS releases: 3.3, 3.0.
+At this time, we support the following LTS releases: 3.3.
 
 
 | LTS         | Supported until |
 | ----------- | --------------- |
-| 3.0         |  2022-07        |
```
[the remark this answers is not in the document]
Comment (mzedel) [reply 302.Release-information/01.Release-schedule/docs.md:21]: what would be your threshold for this? should this only ever be the prior lts release/ 2 releases/ ...?

[the remark this answers is not in the document]
Comment (TheMeaningfulEngineer) [reply 302.Release-information/01.Release-schedule/docs.md:21]: I'd say that should be dictated based on the formatting.
As many as doesn't overtake the page, two last unsupported sound like a good number.

[the remark this answers is not in the document]
Comment (lluiscampos) [reply 302.Release-information/01.Release-schedule/docs.md:21]: @TheMeaningfulEngineer I have not yet published this in hosted nor in stable branches, so we can still revert it in master branch with no much hustle.

I don't have an strong opinion, I thought we agreed (as on "silence" = "agreement") [here](https://github.com/mendersoftware/mender-docs/pull/1854#discussion_r974999085). What do you suggest, then? Have them forever? And what about older LTS releases that are not part of the table?

@kacf @mzedel do you have an opinion?

[the remark this answers is not in the document]
Comment (TheMeaningfulEngineer) [reply 302.Release-information/01.Release-schedule/docs.md:21]: @lluiscampos 
Kristian was apparently saying something like `The lifecycle of an LTS release begins on the release date and ends one year later`. I didn't really get what that means or how it affects the table.

In any case I'd hope to have a simplified version on what 

> What do you suggest, then?

Have a reduced table comparable what [Yocto ](https://wiki.yoctoproject.org/wiki/Releases) has.

> Have them forever? And what about older LTS releases that are not part of the table?

Ideally yes, but I'm unclear at what point will it overtake the entire page.
I'm even up for choosing "have them forever starting from 3.0 and look for a change if needed".

[the remark this answers is not in the document]
Comment (lluiscampos) [reply 302.Release-information/01.Release-schedule/docs.md:21]: See https://github.com/mendersoftware/mender-docs/pull/1872

[the remark this answers is not in the document]
Comment (kacf) [reply 302.Release-information/01.Release-schedule/docs.md:21]: > Have them forever? And what about older LTS releases that are not part of the table?

No strong opinion, but until it becomes a space problem I suppose we can just keep all of them? And clean up at some point in the distant future?

**Pull request merged**